core/src/ast/modules.rs

```diff
@@ -2,13 +2,14 @@ use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 
 use quote::ToTokens;
-use syn::{ImplItem, Item, ItemMod};
+use syn::{ImplItem, Item, ItemMod, UseTree};
 
 use super::{CustomType, Method, ModSymbol, OpaqueStruct, Path, Struct, TypeName};
 
 #[derive(Clone, Serialize, Deserialize, Debug)]
 pub struct Module {
     pub name: String,
+    pub imports: Vec<(Path, String)>,
```

Comment:
If we only extract tagged imports, should these just add to the list of declared types?

Reply:
We want to support imports of other modules, so we can't assume that the import always is to a type.